src/app/[locale]/_utils/common.ts

```diff
@@ -39,7 +39,7 @@ export const getAccessStatus = (sessionData: SessionState | null): 'unlocked' |
   sessionData?.access_enabled ? 'unlocked' : 'locked';
 
 export const getWalletStatus = (walletData?: WalletData): 'on' | 'off' =>
```

Comment:
do we have a status value for 'on' ? I see that in the previous version was 'valid'. Anyway, to be on you have to check htat at least `walletData` is defined I guess.

Reply:
I added this util because it isn't related with the wallet status values that are returned by the INFO api, but is related with the mix panel event IO_PROFILE that requires the statuses "on" or "off" for the ITW_status field